src/Type/Accessory/AccessoryArrayListType.php

```diff
@@ -137,6 +137,11 @@ public function unsetOffset(Type $offsetType): Type
 		return new ErrorType();
 	}
 
+	public function flipArray(): Type
+	{
+		return $this;
```

Comment:
Shouldn't flipping a list degrade to a regular array?

Reply:
oh right, thx! I'll add a test and adapt it

Reply:
adapted